workflows/maskrcnn-training/template.yaml

```diff
@@ -1,26 +1,20 @@
 arguments:
   parameters:
     - name: cvat-annotation-path
-      value: annotation-dump/sample_dataset
+      value: 'artifacts/{{workflow.namespace}}/annotations/'
       hint: Path to annotated data in default object storage. In CVAT, this parameter will be pre-populated.
       displayName: Dataset path
       visibility: internal
 
-    - name: cvat-output-path
-      value: workflow-data/output/sample_output
-      hint: Path to store output artifacts in default object storage. In CVAT, this parameter will be pre-populated.
-      displayName: Workflow output path
-      visibility: internal
-
     - name: cvat-finetune-checkpoint
       value: ''
-      hint: Select the last fine-tune checkpoint for this model. It may take up to 5 minutes for a recent checkpoint show here. Leave empty if this is the first time you're training this model.
+      hint: Path to the last fine-tune checkpoint for this model in default object storage. Leave empty if this is the first time you're training this model.
       displayName: Checkpoint path
       visibility: public
 
     - name: cvat-num-classes
       displayName: Number of classes
-      hint: Number of classes + 1 for background. In CVAT, this parameter will be pre-populated.
+      hint: Number of classes. In CVAT, this parameter will be pre-populated.
       value: '11'
```

Comment:
Minor, but can you set this to an even field like `10`? That way they don't assume +1 for background

Reply:
Done! :+1: